scripts/sas-trading.js

```diff
@@ -924,11 +938,18 @@ class SasTradingMenu extends FormApplication {
                     id: this.TABS.GATHER_INFO,
                     title: `${SasTrading.LANG}.${SasTrading.MENU.TRADE}.${SasTrading.MENU.TRADE_GATHER_INFO}.tab-title`,
                     template: SasTrading.TEMPLATES.PARTIALS.MENU_GATHER_INFO
+                },
+                {
+                    id: this.TABS.BUY_SELL,
+                    title: `${SasTrading.LANG}.${SasTrading.MENU.TRADE}.${SasTrading.MENU.TRADE_BUY_SELL}.tab-title`,
+                    template: SasTrading.TEMPLATES.PARTIALS.MENU_BUY_SELL
                 }
             ],
             activeTab: this.TABS.OVERVIEW,
             selectedCity: selectedCity,
+            selectedGoodName: selectedGoodName,
             selectedGood: selectedGood,
+            selectedBuy: false
```

Comment:
Think it makes sense to have the leftmost radio button selected by default, in this case buy should be true.

Reply:
Done